src/decision/include/LidarDecision.h

```diff
@@ -63,21 +77,21 @@ class LidarObstacle {
      *
      * @return the angle of the Obstacle from the robot
      */
-    float getAvgAngle();
+    angle_t getAvgAngle();
 
     /**
      * Gets the minimum angle from of an object from the robot
      *
      * @ return the minimum angle of the obstacle from the robot
```

Comment:
Just realized that I messed up when I made this. While you're updating this, would you mind replacing `@ return` with `@return` throughout the file? Sorry about that! :disappointed: